src/api/index.mdx

```diff
@@ -14,3 +14,4 @@ The current version of the web API is known as **v0** and is considered to be in
 - [Authentication](/api/auth/)
 - [Permissions](/api/permissions/)
 - [Pagination](/api/pagination/)
+- [Rate Limits](/api/ratelimits)
```

Comment:
This is a bit outside the scope of this PR, but does anyone know why this list here doesn't follow the same order as these left hand navigation? Thinking these should match.